Core/Core/Data/Model/Data_Dashboard.swift

```diff
@@ -134,9 +131,6 @@ public extension DataLayer {
             case courseImage = "course_image"
             case courseAbout = "course_about"
             case courseSharingUtmParameters = "course_sharing_utm_parameters"
-            case courseUpdates = "course_updates"
-            case courseHandouts = "course_handouts"
-            case discussionURL = "discussion_url"
```

Comment:
@miankhalid in the production app, we are using `course_handouts` to show/add the Handouts in the course menu and likewise, we are using `discussion_url` to enable/disable course discussions and `course_updates` being used for course announcements. e.g if `discussion_url` is nil, discussion tab won't be added in the course menu on course dashboard screen.

Should we remove these or keep them and create a ticket to get parity with the production app?

Reply:
@volodymyr-chekyrta the only downside I'm seeing with this approach is we're fattening the response of the said API. Currently, there are courses on our side whose response for Blocks has reached size close to 2 to 3Mbs.

If we keep on adding stuff, it'll only increase further. Plus, Glib will have to update his PR https://github.com/openedx/edx-platform/pull/34273# even further.

For now, I do think we can update this API but in the longer run we should look into splitting these responses into atleast 2 APIs where:

1. one would just return Blocks
2. and the other one would return all the course related details

thoughts?

Reply:
Sure, sounds reasonable 💯 👍! 
But I'd like to check out the size and speed difference before we make any decisions.

Reply:
yep that's why I've asked if Glib can do some performance & load testing before and after the change in his PR.